### PR TITLE
create pools needed for cephfs

### PR DESCRIPTION
**Pull request by mingfang**

### CODE REVIEW

File: mds/entrypoint.sh [hunk 1]
```diff
@@ -18,6 +18,9 @@ fi
 
 # Check to see if we are a new MDS
 if [ ! -e /var/lib/ceph/mds/ceph-$MDS_NAME/keyring ]; then
+   ceph osd pool create cephfs_data 8
```
Comment (leseb) [inline mds/entrypoint.sh]: :-1: 
I believe pool pg and pgp should be configurable.

**Comment by leseb** [comment]
I just thought about it but this is only necessary for release > 0.87.
I'll need to rework a bit the demo container as well.

**Comment by Ulexus** [comment]
Since the master branch is currently based on Giant, this should be "safe," but one of the advantages of the change is to allow flexibility of the data and metadata pools (even more importantly, to allow for multiple metadata services, for multiple CephFS systems on the same cluster...down the road).  At a minimum, whatever solution we implement here must be able to support custom naming of the data and metadata pools.

I think I'd like to take this as more of a wait-and-see thing, though, since this area of Ceph appears to be in a bit of flux at the moment.

**Comment by leseb** [comment]
What about 4 variables then?

* $cephfs_data_pool
* $cephfs_data_pool_pg
* $cephfs_metadata_pool
* $cephfs_metadata_pool_pg

**Comment by Ulexus** [comment]
Works for me

**Comment by leseb** [comment]
Cool, let's wait for @mingfang now. :)

**Comment by mingfang** [comment]
That works for me too. 
Thanks.

**Comment by Ulexus** [comment]
PR #74 Implements this, mostly as described.

**Pull request closed without merging**